extensions/apps/extensions/src/components/pages/extension-edit-page/contributors-page.tsx

```diff
@@ -188,39 +188,51 @@ export const ExtensionEditContributorsPage: React.FC<ExtensionEditContributorsPa
               </Text>
             </Stack>
           )}
-          {showSuggestions && contributors?.length > 0 && searchValue?.length > 1 && (
+          {showSuggestions && searchValue?.length > 1 && (
             <Stack direction="row" customStyle={'relative z-10'}>
               <Card
                 padding={0}
                 radius={20}
                 elevation="2"
                 customStyle="absolute max-h-96 w-full overflow-y-auto scrollbar"
               >
-                <Stack direction="column" spacing="gap-2">
-                  {contributors?.map((profile, index) => (
-                    <Button key={index} plain onClick={() => handleAddContributor(profile)}>
-                      <Stack
-                        padding={16}
-                        direction="row"
-                        justify="between"
-                        align="center"
-                        customStyle="dark:hover:bg-tertiaryDark light:hover:bg-tertiaryLight"
-                      >
-                        <ProfileAvatarButton
-                          profileId={profile?.did?.id}
-                          label={profile?.name}
-                          avatar={transformSource(profile?.avatar?.default)}
-                          alternativeAvatars={profile?.avatar?.alternatives?.map(alternative =>
-                            transformSource(alternative),
-                          )}
-                        />
-                        {addedContributors?.some(
-                          contrib => contrib?.did?.id === profile?.did?.id,
-                        ) && <Icon icon={<CheckIcon />} accentColor />}
-                      </Stack>
-                    </Button>
-                  ))}
-                </Stack>
+                {contributors?.length === 0 && (
+                  <Stack padding={16} spacing="gap-y-2">
+                    <Text variant="body2" weight="bold">
+                      {t('No matches found.')}
+                    </Text>
+                    <Text variant="body2">
+                      {t('To add a user as a contributor, you should follow each other.')}
+                    </Text>
+                  </Stack>
```

Comment:
Is the color the same as whats on the design ? I saw `grey4` for light theme and `grey6` for dark.

Reply:
good catch, thanks